Implementation/Custom-Routing/index.md

```diff
@@ -23,7 +23,7 @@ See: [IContentFinder documentation](../../Reference/Routing/Request-Pipeline/ICo
 
 ### Last Chance IContentFinder
 
-A `IContentLastChanceFinder` is a special implementation of an `IContentFinder` for use with handling 404's. You can implement one of these plugins to decide which Umbraco content page you would like to show when the URL hasn't matched a Umbraco content node.
+A `IContentLastChanceFinder` is a special implementation of an `IContentFinder` for use with handling 404's. You can implement one of these plugins to decide which Umbraco content page you would like to show when the URL hasn't matched a Umbraco content node. When creating packages, or when using class libraries the `SetContentLastChanceFinder` is apart of the `Umbraco.Cms.Web.Website` package.
```

Comment:
@eshanrnh what I meant was that when you have a package or a class library you'll need to include the package reference `		<PackageReference Include="Umbraco.Cms.Web.Website" Version="9.5.2" />` in order to be able to use this extension method.

Reply:
Thanks for getting back to us @royberris , and for help clearing this up!

So, as what you're adding here is very specific to packages, I think it would be a good idea to add it as a note, instead of part as a paragraph.

If it's a note it will stand out, and people will be able to easily find it - or ignore it, if they are not creating a package.

What do you think of that?
We have some docs on how to add notes here: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips

Reply:
Hi @sofietoft I updated the PR